Assets/Scripts/Spawner/impl/ImmobileSpawner.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Creatures.Api;
+using Spawner.Api;
+
+namespace Spawner.impl {
+    public class ImmobileSpawner<T> : CreatureSpawner<T> where T : Creature
```

Comment:
So maybe this is a `SpawnerAction` then yeah? that makes sense more, right now it sounds like these things are doing the spawning

Reply:
Yeah, this should be `SpawnAction`

Reply:
updated